Sources/SPMBuildCore/BuildParameters/BuildParameters.swift

```diff
@@ -182,13 +94,22 @@ public struct BuildParameters: Encodable {
     public var pkgConfigDirectories: [AbsolutePath]
 
     /// The architectures to build for.
+    // FIXME: this may be inconsistent with `targetTriple`.
     public var architectures: [String]?
 
     /// How many jobs should llbuild and the Swift compiler spawn
     public var workers: UInt32
 
     /// If should link the Swift stdlib statically.
-    public var shouldLinkStaticSwiftStdlib: Bool
+    @available(*, deprecated, renamed: "linkingParameters.shouldLinkStaticSwiftStdlib")
```

Comment:
I don't think we need any of the compatibility layer, `BuildParameters` is not supposed to be a type that is used outside SwiftPM itself.